modules/misc/news.nix

```diff
@@ -1132,7 +1132,15 @@ in
         time = "2023-06-17T22:18:22+00:00";
         condition = config.programs.zsh.enable;
         message = ''
-          A new modules is available: 'programs.zsh.antidote'
+          A new module is available: 'programs.zsh.antidote'
+        '';
+      }
+
+      {
+        time = "2023-06-30T14:46:22+00:00";
+        condition = config.services.ssh-agent.enable;
```

Comment:
If I understand correctly, this will now only render the news item for people that already have the service enabled.

Reply:
Yeah, I assume this is a mistake. My guess is that it should be `condition = hostPlatform.isLinux;` and the module itself should contain

```
assertions = [
  (lib.hm.assertions.assertPlatform "services.ssh-agent" pkgs lib.platforms.linux)
];
```

since it is Linux-only.